cves/kernel/CVE-2021-3640.yml

```diff
@@ -480,11 +502,10 @@ CWE:
 - '416'
 - '362'
 CWE_note: |
-  CWE as registered in the NVD. If you are curating, check that this
-  is correct and replace this comment with "Manually confirmed".
+  Manually confirmed.
 nickname_instructions: |
   A catchy name for this vulnerability that would draw attention it.
   If the report mentions a nickname, use that.
   Must be under 30 characters. Optional.
-nickname:
+nickname: 
```

Comment:
How about "Locking Inquisition" a reference to how "Nobody expects the Spanish Inquisition" this is issue was caused by an unexpected locking behavior (https://bugzilla.redhat.com/show_bug.cgi?id=1980646)